lib/index.ts

```diff
@@ -88,12 +89,15 @@ function extractTargetFrameworksFromFiles(
 
   const manifestFileContents = fs.readFileSync(manifestFileFullPath, 'utf-8');
   const manifestFileExtension = path.extname(manifestFileFullPath);
+  const manifestFileName = path.basename(manifestFileFullPath);
 
-  if (_.includes(PROJ_FILE_EXTENSION, manifestFileExtension)) {
+  if (_.includes(PROJ_FILE_EXTENSIONS, manifestFileExtension)) {
     return extractTargetFrameworksFromProjectFile(manifestFileContents);
+  } else if (manifestFileName === 'packages.config') {
```

Comment:
you can use _.endsWith() as we do in previous function, line 67

Reply:
I've made that change now.